client/me/purchases/manage-purchase/index.jsx

```diff
@@ -353,7 +360,7 @@ const ManagePurchase = React.createClass( {
 			</span>
 		);
 
-		if ( isLoading || ! showEditPaymentDetails( purchase ) ) {
+		if ( isLoading || ! showPaymentDetails( purchase ) ) {
```

Comment:
Also what were the reasons for not making these two functions part of the `ManagePurchase` component?